packages/components/src/presentation-components/common/ContentBuilder.ts

```diff
@@ -139,9 +139,10 @@ class StructMembersAppender implements INestedPropertiesAppender {
     this._members[record.fieldHierarchy.field.name] = record.record;
   }
   public finish(): void {
+    const sortedMembers = Object.entries(this._members).sort();
     const value: StructValue = {
       valueFormat: UiPropertyValueFormat.Struct,
-      members: this._members,
+      members: Object.fromEntries(sortedMembers),
```

Comment:
Can you add test to check properties order?

Reply:
Added.